src/patternfly/components/Tabs/tabs.scss

```diff
@@ -220,6 +240,33 @@
     border: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
   }
 
+  &.pf-m-secondary {
+    &::before {
```

Comment:
Can you wrap the block that defines `::before` above in `&:not(.pf-m-secondary)` so you don't have a pseudo element at all on the secondary buttons? It's better to do that than overwrite properties if you can.